src/styles/_class-overrides.scss

```diff
@@ -0,0 +1,525 @@
+            @include u-bg('accent-cool-lighter');
+            border: none;
+          }
+        }
+        //links and nav label buttons in the nav menu
+        //styling applies when accordion buttons are open and closed
+        a,
+        .usa-accordion__button[aria-expanded='false'],
+        .usa-accordion__button[aria-expanded='true'] {
+          @include hover-border(
+            'left',
+            'secondary',
+            'base-darker',
+            'accent-cool-lighter'
+          );
+          @include u-text('base-darker');
+          @include u-bg('accent-cool-lighter');
+          padding: 0 units(0.5);
+          margin: units(1.5) units(2);
+          &:hover {
+            @include u-text('base-darker');
+            @include u-bg('accent-cool-lighter');
+            @include u-text('bold');
+            padding: 0 units(0.5);
+            margin: units(1.5) units(2);
+          }
+        }
+      }
+    }
+  }
+
+  //header in mobile view
+  .usa-navbar {
+    @include flex-center-space-between();
+    margin: units(4) 0 units(2) 0;
+    //smaller logo size
+    .header__logo {
+      width: units('card');
+      height: auto;
+      padding-left: units(2);
+    }
+    //nav menu button
+    .usa-menu-btn {
+      @include u-bg('secondary');
+    }
+  }
+
+  //languages in nav bar
+  .header__language-nav-container {
+    //position languages at the bottom
+    position: absolute;
+    bottom: 0;
+    @include u-bg('base-dark');
+    padding: units(1) 0;
+    //right align right to left languages
+    .header__language-nav-RtoL {
+      display: flex;
+      align-items: center;
+      justify-content: flex-end;
+      flex-direction: row;
+      flex-wrap: wrap;
+    }
+    //left align left to right languages
+    .header__language-nav-LtoR {
+      display: flex;
+      align-items: center;
+      justify-content: flex-start;
+      flex-direction: row;
+      flex-wrap: wrap;
+    }
+    //language button styling
+    .header__language-nav-items {
+      .usa-button--unstyled {
+        @include u-text('accent-cool-lighter');
+        &:hover {
+          @include u-text('primary');
+          @include u-bg('base-dark');
+        }
+      }
+    }
+  }
+}
+
+//desktop view
+@media (min-width: units('desktop')) {
+  //show language selector button and project title
+  .banner__language-selector,
+  .header__title {
+    display: block;
+  }
+  .usa-nav {
+    margin-bottom: units(2);
+  }
+  .header__logo-title {
+    @include flex-center-space-between();
+  }
+  .usa-navbar {
+    .header__logo {
+      width: units('card-lg');
+      height: auto;
+    }
+  }
+
+  .usa-nav {
+    padding: units(2);
+    .usa-nav__inner {
+      .usa-nav__primary-item {
+        .usa-nav__submenu {
+          @include u-bg('base-darker');
+          .usa-nav__submenu-item {
+            @include u-bg('base-darker');
+            a {
+              @include u-text('accent-cool-lighter');
+              @include u-bg('base-darker');
```

Comment:
`@include u-bg('base-darker')` is repeated multiple times, does it have to be?

Reply:
As things are now, if I only set the background color on 455 then the backgrounds in the other nested elements are white, since they were set to that color in the mobile view. I think each has to be reset for desktop view?

Reply:
I see.

Reply:
I tried to use `@extend`, but got this error:
```
You may not @extend an outer selector from within @media.
```